packages/theming/src/index.ts

```diff
@@ -13,4 +13,5 @@ export { default as retrieveComponentStyles } from './utils/retrieveComponentSty
 export { default as withTheme } from './utils/withTheme';
 export { default as getDocument } from './utils/getDocument';
 export { default as getColor } from './utils/getColor';
```

Comment:
I'm not seeing the `getLineHeight` export that you've added in this PR. Are we missing an export?